client/components/editor/structure/AddElement/SelectElement.vue

```diff
@@ -86,7 +86,7 @@ export default {
       if (outlineActivity || examGroup) return ['TR'];
     },
     columnWidth() {
-      return `col-xs-${12 / this.columns}`;
+      return `col-xs-${Math.round(12 / this.columns)}`;
```

Comment:
```suggestion
      return `col-xs-${Math.floor(12 / this.columns)}`;
```
Let's go with rounding down instead to prevent potential overflow. ☝️